modules/phase_field/tests/MultiPhase/DerivativeMultiPhaseMaterial_SwitchingMix.i

```diff
@@ -0,0 +1,385 @@
+[Mesh]
```

Comment:
model you test based on `switchingfunction3phasematerial.i` for example. You do not need a solve to test this material. Please try to keep to the standard for filename capitalization and use of underscores for at least the directory the file is located in.